components/profile_popover/profile_popover.tsx

```diff
@@ -617,31 +619,50 @@ class ProfilePopover extends React.PureComponent<ProfilePopoverProps, ProfilePop
             </OverlayTrigger>
         );
 
-        const startCallMessage = formatMessage({
-            id: t('webapp.mattermost.feature.start_call'),
-            defaultMessage: 'Start Call',
-        });
-        const callButton = (
-            <OverlayTrigger
-                delayShow={Constants.OVERLAY_TIME_DELAY}
-                placement='top'
-                overlay={
-                    <Tooltip id='startCallTooltip'>
-                        {startCallMessage}
-                    </Tooltip>
-                }
-            >
-                <button
-                    type='button'
-                    className='btn icon-btn'
+        const renderCallButton = () => {
+            if (!this.props.isCallsEnabled) {
+                return null;
+            }
+            const disabled = this.props.isUserInCall || this.props.isCurrentUserInCall;
+            const startCallMessage = this.props.isUserInCall ? formatMessage({
+                id: t('user_profile.call.userBusy'),
+                defaultMessage: 'User is in another call',
+            }) : formatMessage({
+                id: t('webapp.mattermost.feature.start_call'),
+                defaultMessage: 'Start Call',
+            });
+            const iconButtonClassname = classNames('btn icon-btn', {'icon-btn-disabled': disabled});
```

Comment:
I couldnt use `disabled` prop for button. Because when I use, for some reason tooltip is not shown when the button is disabled

Reply:
Thanks `aria-disabled` is added. `aria-label` is already added to `PhoneInTalkIcon` component. Do you think it is still necessary add `aria-label` again to wrapping component (i.e. button with `id='startCallButton'`)

Reply:
You are right, I think that's fine.